### PR TITLE
[FEAT][LIVE-10174][LIVE-10173] add sepolia and holesky testnets

### PR DESCRIPTION
**Pull request by chabroA**
### 📝 Description

Add Ethereum sepolia and holesky testnet integrations.

Sepolia seems to have a token referenced in the CAL so this part is updated as well. Holesky does not.

### ❓ Context

- https://ledgerhq.atlassian.net/browse/LIVE-10174
- https://ledgerhq.atlassian.net/browse/LIVE-10173

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - Ethereum sepolia and holesky testnet available on LLD and LLM (after enabling "developer mode" in "experimental features" settings menu 

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: .github/workflows/bot-staging-explorer-eth.yml [hunk 1]
```diff
@@ -37,5 +37,5 @@ jobs:
           SLACK_API_TOKEN: ${{secrets.SLACK_API_TOKEN}}
           SLACK_ICON_EMOJI: ":bot-seed2:"
           SLACK_CHANNEL: explorer-bot-stg
-          BOT_FILTER_CURRENCIES: avalanche_c_chain,bsc,polygon,ethereum,ethereum_classic,ethereum_goerli
+          BOT_FILTER_CURRENCIES: avalanche_c_chain,bsc,polygon,ethereum,ethereum_classic,ethereum_goerli,ethereum_sepolia,ethereum_holesky
```
Comment (hedi-edelbloute) [inline .github/workflows/bot-staging-explorer-eth.yml:40]: Please make this work on either nitrogen or oxygen

**Review by hedi-edelbloute — COMMENTED** [review]
Bot edits needed

File: .github/workflows/bot-staging-explorer-eth.yml (hunk 1 above)
Comment (hedi-edelbloute) [reply .github/workflows/bot-staging-explorer-eth.yml:40]: We chose to make it run daily, no update needed

**Review by hedi-edelbloute — APPROVED** [review]

File: .github/workflows/bot-staging-explorer-eth.yml (hunk 1 above)
Comment (chabroA) [reply .github/workflows/bot-staging-explorer-eth.yml:40]: To be consistent with what is done for the goerli testnet today, there is nothing to update, which mean that these 2 new testnets will be used for daily nonreg with Nitrogen, as it's already the case today for goerli 👍

**Pull request merged**